docs/website/data/integrations.yaml

```diff
@@ -1067,8 +1067,8 @@ integrations:
       - scalr
     blogs:
       - https://www.scalr.com/blog/opa-is-to-policy-automation-as-terraform-is-to-iac/
-
-  spire:
+      
+    spire:
```

Comment:
This looks like an accidental change.

Reply:
I have fixed this is a subsequent merge.